pages/terms/index.html

```diff
@@ -62,6 +62,11 @@ <h3 class="col-md-7 col-md-offset-5 col-print-12">{{ step.title }}</h3>
           {% endfor %}
         </div>
       </div>
+      <div class="row">
+        <div class="col-xs-12 text-center">
+          <a id="download-docx" class="btn btn-primary" href="Gruntwork-TermsOfService.docx" title="Download editable Gruntwork Terms of Service">Download MS Word Docx</a>
```

Comment:
Should probably add a note that "This document is intended solely for Enterprise Customers, as described in the FAQ" (with a corresponding link).

Reply:
Updated in .